pkg/resizer/csi_resizer.go

```diff
@@ -118,7 +120,13 @@ func (r *csiResizer) Name() string {
 	return r.name
 }
 
+// CanSupport returns whether the PV is supported by resizer
+// Resizer will resize the volume if it is CSI volume or is migration enabled in-tree volume
 func (r *csiResizer) CanSupport(pv *v1.PersistentVolume) bool {
+	if csitranslationlib.IsMigratedCSIDriverByName(r.name) /* && migration enabled*/ {
```

Comment:
shouldn't this still verify if volume being resized matches registered driver name?